h/static/styles/help-page.scss

```diff
@@ -1,10 +1,10 @@
 $base-font-size: 16px;
 $base-line-height: 22px;
 
-@import './reset';
-@import './common';
-@import './elements';
-@import './mixins/icons';
+@import 'reset';
```

Comment:
If in future we move foundational modules such as colors and typography to a separate module published on npm, then I imagine we would add `node_modules/` to the search path and the imports might look like this:

```scss
@import 'hypothesis-frontend-toolkit/colors'

@import './mixins/something-local'
```